### DISCUSSION TITLE
Evaluation using Scale transformation from old data. PaddlePaddle Framework.

### DISCUSSION THREAD

**Discussion by fersebas** [PaddlePaddle/PaddleSpatial#104] [BaiduKDDCup2022] [answered]
HI! I think there is a bug in the evaluation script or need to update

https://github.com/PaddlePaddle/PaddleSpatial/blob/main/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation/paddlepaddle/evaluation.py
![image](https://user-images.githubusercontent.com/49498032/170301976-5a4c287c-2c88-47f1-9c8e-214f9df4faf0.png)

Why the filename is fixed to the old one "sdwpf_baidukddcup2022_full.csv".

So  when we are forecasting in this line

    train_data = exp.load_train_data()

We read the old one csv and get scaler with this old data. Please check it out.

**Comment by zhoujingbo** [comment, marked as answer]
Hi, 

Thanks for pointing out this. 

The corrected file name should be "wtbdata_245days.csv". We have updated the [evaluation.py](https://github.com/PaddlePaddle/PaddleSpatial/blob/main/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation/paddlepaddle/evaluation.py) file accordingly. 

Thanks a lot.

Regards

**Reply by fersebas** [reply]
Thanks :)
